docs/welcome/index.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 1
+slug: /welcome
+---
+
+# Welcome
+
+Welcome to the *Microprocessor Architecture* class.
+
+During the class you will learn how to design and implement a hardware device. 
+
+You will learn:
+  - how a microporcessor (*MCU*) works, using as an example the RP2040 MCU from the [Raspberry Pi Pico W](https://www.raspberrypi.com/products/raspberry-pi-pico/);
+  - how to design your hardware schematics using [KiCad](https://www.kicad.org/) and how to create the bill of materials (*BOM*) for it;
+  - how to develop the software that controlls hardware in [Rust](https://www.rust-lang.org/).
+
+
+## Team
+
+### Alexandru Radovici
+![Alexandru Radovici](images/alexandru_radovici.jpg)
+
+Course Professor \
+GitHub: [alexandruradovici](https://github.com/alexandruradovici) \
+Email: alexandru.radovici@upb.ro
+
+### Irina Niță
+![Irina Nita](images/irina_nita.jpg)
+ 
+Lab Professor \
+GitHub: [irina-nita](https://github.com/irina-nita) \
+Email: irina.nita@oxidos.io
+
+### Irina Bradu
+![Irina Bradu](images/irina_bradu.jpg)
+ 
+Lab Professor \
+GitHub: [Irina Bradu](https://github.com/irina-b-dev) \
+Email: irina.bradu@wyliodrin.com
+
+### Dănuț Aldea
+![Danut Aldea](images/danut_aldea.jpg)
+ 
+Lab Professor \
+GitHub: [danutaldea](https://github.com/danutaldea) \
+Email: danut.aldeaa@oxidos.io
+
+### Andrei Zamfir
+![Andrei Zamfir](images/andrei_zamfir.jpg)
+ 
+Lab Professor \
+GitHub: [ZamfirAndreiPaul](https://github.com/ZamfirAndreiPaul) \
+Email: andrei.zamfir@wyliodrin.com
+
+### Teodor Dicu
+![Teodor Dicu](images/teodor_dicu.jpg)
+ 
+Lab Professor \
+GitHub: [DTeodor-Alexaandru](https://github.com/DTeodor-Alexaandru) \
+Email: teodor.dicu@wyliodrin.com
+
+## Timetable
+
+| Group | Day | Hour | Room |
+|-------|------|-------|-------|
+| Lecture | Friday | 10 - 12 | AN024 |
+| IoT | Thursday | 10 - 12 | ED 217 |
+| 1221A 1/2 | Thursday | 8 - 10 | ED 218 |
+| 1221A 2/2 | Thursday | 8 - 10 | ED 217 |
+| 1221B 1/2 | Thursday | 10 - 12 | ED 218 |
+| 1221B 2/2 | Thursday | 14 - 16 | ED 217 |
+| 1222A | Thursday | 14 - 16 | ED 218 |
+| 1222B | Thursday | 12 - 14 | ED 218 |
+
+## Grading
+
+| Part | Description | Points |
+|--------|-------------|--------|
+| [Lecture](/docs/category/lecture) tests | You will have a test at every class with subjects from the previous class. | 2p |
+| [Lab](/docs/category/lab) | Your work at every lab will be graded. | 2p |
+| [Project](/docs/project) | You will have to design and implement a hardware device. Grading will be done for the documentation, hardware design and software development. | 5p |
```

Comment:
+ Also PM fair
Proposal: each milestone completed 1p and PM Fair 2p

Reply:
My idea is to add those in the project description.